crates/swc_ecma_minifier/tests/fixture/issues/typescript/1/output.js

```diff
@@ -19,48 +19,37 @@ var ts, ts1, dynamicImportUMDHelper;
                 case 79 /* Identifier */ :
                     return substituteExpressionIdentifier(node);
                 case 207 /* CallExpression */ :
-                    return function(node) {
-                        if (ts1.isIdentifier(node.expression)) {
-                            var expression = substituteExpressionIdentifier(node.expression);
-                            if (noSubstitution[ts1.getNodeId(expression)] = !0, !ts1.isIdentifier(expression) && !(4096 /* HelperName */  & ts1.getEmitFlags(node.expression))) return ts1.addEmitFlags(factory.updateCallExpression(node, expression, /*typeArguments*/ void 0, node.arguments), 536870912 /* IndirectCall */ );
-                        }
-                        return node;
-                    }(node);
+                    if (ts1.isIdentifier(node.expression)) {
+                        var expression = substituteExpressionIdentifier(node.expression);
+                        if (noSubstitution[ts1.getNodeId(expression)] = !0, !ts1.isIdentifier(expression) && !(4096 /* HelperName */  & ts1.getEmitFlags(node.expression))) return ts1.addEmitFlags(factory.updateCallExpression(node, expression, /*typeArguments*/ void 0, node.arguments), 536870912 /* IndirectCall */ );
+                    }
+                    break;
                 case 209 /* TaggedTemplateExpression */ :
-                    return function(node) {
-                        if (ts1.isIdentifier(node.tag)) {
-                            var tag = substituteExpressionIdentifier(node.tag);
-                            if (noSubstitution[ts1.getNodeId(tag)] = !0, !ts1.isIdentifier(tag) && !(4096 /* HelperName */  & ts1.getEmitFlags(node.tag))) return ts1.addEmitFlags(factory.updateTaggedTemplateExpression(node, tag, /*typeArguments*/ void 0, node.template), 536870912 /* IndirectCall */ );
-                        }
-                        return node;
-                    }(node);
+                    if (ts1.isIdentifier(node.tag)) {
+                        var tag = substituteExpressionIdentifier(node.tag);
+                        if (noSubstitution[ts1.getNodeId(tag)] = !0, !ts1.isIdentifier(tag) && !(4096 /* HelperName */  & ts1.getEmitFlags(node.tag))) return ts1.addEmitFlags(factory.updateTaggedTemplateExpression(node, tag, /*typeArguments*/ void 0, node.template), 536870912 /* IndirectCall */ );
+                    }
+                    break;
```

Comment:
Is this correct?
Original code is returning.

Reply:
Well, this is cool.

The original code
```
switch ...
  case ...:
      return substituteCallExpression(node)

return node
```
with function inline and invoke it becomes
```
switch ...
  case ...:
      ...
      return node

return node
```
then with if_return or something alike
```
switch ...
  case ...:
      ...
      break

return node
```

Reply:
TLDR it's the result of combined optimization,

Reply:
Ah, I see. Thank you!

Reply:
I really agree that this is cool 👍